app/src/main/java/com/github/se/orator/ui/friends/FriendsScreen.kt

```diff
@@ -212,6 +216,9 @@ fun FriendItem(friend: UserProfile) {
                 overflow = TextOverflow.Ellipsis,
                 modifier = Modifier.testTag("friendBio#${friend.uid}"))
           }
+          Spacer(modifier = Modifier.weight(1f)) // Pushes the delete button to the right
```

Comment:
The DeleteFriendButton composable is integrated well within the FriendItem. It’s smartly positioned to the right, making it accessible without crowding the UI.